arcanist-user.rst

```diff
@@ -243,10 +243,25 @@ shows the current stack of revisions:
    :align: center
    :alt: Screenshot of a revision stack
 
-You can also add ``Depends on D<revision ID>`` to the child's commit
-message, replacing ``<revision ID>`` with the ID of the parent
+You can also add ``Depends on D<revision ID>`` to the child's revision
+summary, replacing ``<revision ID>`` with the ID of the parent
 revision. (This needs to be its own paragraph, separated by a blank line.)
-The relationship will be created when ``arc diff`` is run.
+The relationship will be created when ``arc diff`` is run.::
+
+    ... commit comment ...                                                                                                                                                                                                                                                       
```

Comment:
This is actually the commit summary, i.e. the first line of the commit message. The rest of the comment will go into the Summary field below.